scripts/E2ETestDeploy.s.sol

```diff
@@ -95,6 +95,10 @@ contract E2ETestDeploy is Script, IICS07TendermintMsgs, DeploySP1ICS07Tendermint
         json.serialize("ics07Tendermint", Strings.toHexString(address(ics07Tendermint)));
         json.serialize("ics26Router", Strings.toHexString(address(ics26Router)));
         json.serialize("ics20Transfer", Strings.toHexString(address(ics20Transfer)));
+        json.serialize("ibcERC20Logic", Strings.toHexString(address(ibcERC20Logic)));
+        json.serialize("verifier", Strings.toHexString(address(verifier)));
+        json.serialize("routerProxy", Strings.toHexString(address(routerProxy)));
```

Comment:
The routerProxy and transferProxy will always be the same as ics26Router and ics20Transfer, so probably not necessary to output those. Verifier I can see the use for.

Reply:
Oh yea good point, just confirmed:


```
== Return ==
0: string "{\"erc20\":\"0x48fd1226d797400779bbfaf1706f5fb8da04ae91\",\"ibcERC20Logic\":\"0xe53275a1fca119e1c5eeb32e7a72e54835a63936\",\"ics07Tendermint\":\"0xcb1dc4af63cfdaa4b9bff307a8dd4dc11b197e8f\",\"ics20Transfer\":\"0x7e7ad18adc99b94d4c728fdf13d4de97b926a0d8\",\"ics26Router\":\"0x70ac5980099d71f4cb561bbc0fcfef08aa6279ec\",\"routerProxy\":\"0x70ac5980099d71f4cb561bbc0fcfef08aa6279ec\",\"transferProxy\":\"0x7e7ad18adc99b94d4c728fdf13d4de97b926a0d8\",\"verifier\":\"0x2854cfac53fcab6c95e28de8c91b96a31f0af8dd\"}"
```

Will remove the routerProxy and transferProxy